rebar.lock

```diff
@@ -0,0 +1 @@
+[{<<"jsx">>,{pkg,<<"jsx">>,<<"2.8.0">>},0}].
```

Comment:
it's a auto generated file from rebar3, it guarantees that builds are repeatable as in rebar will look at the .lock file to determine the versions, that allows using something like just `jsx` as a dependency